packages/client/hmi-client/src/components/widgets/tera-tab-group.vue

```diff
@@ -2,7 +2,25 @@
 	<nav :style="{ '--nb-tabs': tabs.length }">
 		<header v-for="(tab, index) in tabs" :key="index">
 			<div class="tab" @click="emit('select-tab', tab)" :active="activeTabIndex === index">
-				<i :class="iconClassname(tab.assetType ?? null)" />
+				<template v-if="typeof getAssetIcon(tab.assetType ?? null) === 'string'">
+					<i
+						v-if="getAssetIcon(tab.assetType ?? null).substring(0, 3) === 'pi '"
+						:class="getAssetIcon(tab.assetType ?? null)"
+					/>
+					<vue-feather
+						v-else
+						:type="getAssetIcon(tab.assetType ?? null)"
+						size="1rem"
+						stroke="rgb(16, 24, 40)"
+					/>
+				</template>
+				<component
+					v-else
+					:is="getAssetIcon(tab.assetType ?? null)"
+					class="p-button-icon-left"
+					fill="rgb(16, 24, 40)"
+					style="overflow: visible"
+				/>
```

Comment:
Okay this is too much, it would be simpler to use only _Feather Icons_ for all assets, and stop using _Prime Icons_ `pi pi-*` to avoid this kafkaesque rendering.
I know you spend a good amount of time on this solution, and it's nice, but this is too complex just for some icons.